src/hash.rs

```diff
@@ -0,0 +1,22 @@
+use std::hash::Hasher;
+
+pub trait ContentHasher: Send + Sync {
+    fn hash(&self, content: &str) -> u64;
+}
+
+#[derive(Debug, PartialEq, Eq, Clone, Copy)]
+pub struct FxHasher;
+
+impl FxHasher {
+    pub fn new() -> Self {
+        Self
+    }
+}
+
+impl ContentHasher for FxHasher {
+    fn hash(&self, content: &str) -> u64 {
+        let mut hasher = rustc_hash::FxHasher::default();
+        hasher.write(content.as_bytes());
+        hasher.finish()
+    }
+}
```

Comment:
let's have this as just a function actually, i don't see why we would need a different implementation ( at least for now )

Reply:
If one day we start using async I/O in Forest, something like this would be useful. But, yeah, for now we can go with just a function. =D